lib/fakefs/file.rb

```diff
@@ -280,14 +280,14 @@ def self.executable?(filename)
     def self.chown(owner_int, group_int, filename)
       file = FileSystem.find(filename)
       if owner_int && owner_int != -1
-        owner_int.is_a?(Fixnum) || fail(TypeError,
-                                        "can't convert String into Integer")
-        file.uid = owner_int
+        owner_int.respond_to?(:to_i) || fail(TypeError,
```

Comment:
```
''.respond_to?(:to_i)
=> true
```

... if this did not break anything maybe add the missing test ... can check for `is_a?(Integer)` though ...